src/tests/audio/tasks/speech_enhancement_test.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for the speech enhancement task."""
+import os
+
+import pytest
+from speechbrain.inference.separation import SepformerSeparation as separator
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.audio.tasks.preprocessing.preprocessing import resample_audios
+from senselab.audio.tasks.speech_enhancement.api import enhance_audios
+from senselab.audio.tasks.speech_enhancement.speechbrain import SpeechBrainEnhancer
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.model import HFModel
+
+
+@pytest.fixture
+def sample_audio() -> Audio:
```

Comment:
might be better to make a utility function for the tests that does this so we don't have to keep repeating it

Reply:
Done